subsys/net/l2/openthread/Kconfig

```diff
@@ -265,6 +265,12 @@ config OPENTHREAD_NUM_MESSAGE_BUFFERS
 	help
 	  "The number of message buffers in the buffer pool."
 
+config OPENTHREAD_MESSAGE_BUFFER_SIZE
+	int "The size of a message buffer in bytes"
+	default 128
```

Comment:
Existing applications that pull in this change, might not work as expected? I like the option, just not sure if we should reduce it as default.

Reply:
Default value in OpenThread is 128 bytes for 32bit architectures and 256 bytes for 64bit architectures. Are you aware of any Kconfig option to detect the system's CPU word length?

In any case static asserts in `message.hpp` and `coap_message.hpp` will detect any issues with the buffer size at compile time.